pkg/kubelet/eviction/api/types.go

```diff
@@ -40,6 +40,8 @@ const (
 	SignalAllocatableMemoryAvailable Signal = "allocatableMemory.available"
 	// SignalAllocatableNodeFsAvailable is amount of local storage available for pod allocation
 	SignalAllocatableNodeFsAvailable Signal = "allocatableNodeFs.available"
+	// SignalPIDAvailable is amount of PID available for pod allocation
+	SignalPIDAvailable Signal = "pid.available"
```

Comment:
Would it make sense to have a node-allocatable amount of PIDs, like we do for other resources?

Reply:
hm..., used to propose this [here](https://github.com/kubernetes/kubernetes/issues/54313#issuecomment-338636646), but I agree with @vishh 's feedback to make it PID pressure, that make it easy to use.